### ISSUE TITLE
Error when using with relay-runtime 11.0.2

### ISSUE THREAD

**Issue by litewarp** [morrys/wora#82]
When using `relay-runtime 11.0.2`, I get:

```
error - ./node_modules/@wora/relay-offline/lib/Environment.js:5:0
Module not found: Can't resolve 'relay-runtime/lib/store/RelayModernQueryExecutor'
```

Downgrading back to `relay-runtime 11.0.1` fixes the error.

**Comment by morrys** [comment]
thanks for the tip.
relay-runtime renamed API https://github.com/facebook/relay/blob/master/packages/relay-runtime/store/OperationExecutor.js

next week I will consider how to release the fix.

**Comment by morrys** [comment]
Hi @litewarp, issue fixed with [react-relay-offline 4.0.0 version](https://github.com/morrys/react-relay-offline/releases/tag/v4.0.0)
